Newsweek

Dflexcee opened the pull request:
This is my Project: NexsWEEK html/css project
This was done alone by me followed by the instruction of the success student. This is due to the absence of my coding partner. i have been paired twice and none of them did not show up. So Willio sent me A mail to continue with HTML because currently no one to be paired me with again.

Nevertheless, my previous projects has been approved and i did it alone. But i was advice by last reviewer to add this go Ahead instruction in my PR.
Thanks for your understanding.

Dflexcee committed:
this is the new newsweek to accomplish the errors i have in my formal news projects. i was having a terrible responsiveness errors. i have to start up the page at fesh.

Dflexcee committed:
Update .stickler.yml

Dflexcee committed:
Update README.md

i corrected my read me

Dflexcee committed:
Update README.md

corrected read me

MauricioRobayo commented on README.md:
```diff
@@ -1 +1,24 @@
+#### NEWSWEEK-CLONE
+
+### Authors:
+d[FLEXCEE](https://github.com/Dflexcee/)  
```
You left the letter `d` outside the square brackets 🤭.

MauricioRobayo reviewed (changes requested):
Congratulations, the is looking great 🔥 .

I can see you fixed the issues pointed out on the previous PR: https://github.com/Dflexcee/NEWSWEEK/pull/1

The project is really coming to a good end, this is great execution, truly admire your perseverance and the effort you have put into this 🥇 .

This is so close to approval, there is an issue with your HTML, please [validate it](https://validator.w3.org/nu/?doc=https%3A%2F%2Fraw.githack.com%2FDflexcee%2Fmynewsweek-project%2Fnewsweek%2Findex.html) and fix the issue.

Almost there 😄 .

Keep up the good work and the perseverance 💪 .

**Status**: Request changes.

Dflexcee committed:
Update index.html

i have corrected the errors from html validations.smiles. i remebr you cant have a tag as a descendant of a button

Dflexcee committed:
Update README.md

corrected my name on my readme

MauricioRobayo reviewed (approved):
Awesome!

Remember to include the `.sass-cache` directory in your `.gitignore`, it should not be uploaded into the repo. It seems that I missed that one on the initial review, but please take a look at these links to make sure it won't happen again 😄 :

- https://www.atlassian.com/git/tutorials/saving-changes/gitignore
- https://help.github.com/en/github/using-git/ignoring-files

Please move ahead 🚀 , this is **Approve**.
